modules/Story/Story.tsx

```diff
@@ -30,46 +30,25 @@ function Story({ story }: Props) {
         return null;
     }
 
-    const { title, subtitle, content, format_version, categories, links } = story;
-    const headerImage = story.header_image ? JSON.parse(story.header_image) : null;
-    const hasHeaderImage = Boolean(headerImage);
+    const { categories, links } = story;
     const hasCategories = categories.length > 0;
+    const nodes = JSON.parse(story.content);
 
     return (
         <Layout>
             <StorySeo story={story} noindex={!isAnalyticsEnabled} />
             <article className={styles.story}>
-                <div
-                    className={classNames(styles.container, {
-                        [styles.withImage]: hasHeaderImage,
-                    })}
-                >
+                <div className={styles.container}>
+                    {isEmbargoStory(story) && <Embargo story={story} />}
                     {hasCategories && <CategoriesList categories={categories} showAllCategories />}
-                    <h1 className={styles.title}>{title}</h1>
-                    <p className={styles.subtitle}>{subtitle}</p>
+                    <HeaderRenderer nodes={nodes} />
                     {showDate && (
                         <p className={styles.date}>
                             <StoryPublicationDate story={story} />
                         </p>
                     )}
                     <StoryLinks url={links.short || links.newsroom_view} />
-                    {headerImage && (
-                        <Image
-                            alt=""
-                            className={styles.mainImage}
-                            objectFit="cover"
-                            layout="fill"
-                            imageDetails={headerImage}
-                        />
-                    )}
-                    {isEmbargoStory(story) && <Embargo story={story} />}
-                    {format_version === FormatVersion.HTML && (
-                        // eslint-disable-next-line react/no-danger
-                        <div dangerouslySetInnerHTML={{ __html: content }} />
-                    )}
-                    {format_version === FormatVersion.SLATEJS && (
-                        <SlateRenderer nodes={JSON.parse(content as string)} />
-                    )}
+                    <ContentRenderer nodes={nodes} />
```

Comment:
Now that header image is renderered as part of the content and treated as any other image, it has different margins than the old header image which had some extra spacing around it. Personally I don't think it's a big deal but just letting you know.

You can compare, this is production:
![theme-nextjs-lena-the-good-newsroom vercel app_i-listened-to-animals-hearts-to-reveal-their-hidden-emotional-worlds](https://github.com/prezly/theme-nextjs-greta/assets/4209081/88005a50-ea2a-40c4-9fc7-2c840eec533e)

This is now:
![localhost_3000_i-listened-to-animals-hearts-to-reveal-their-hidden-emotional-worlds (2)](https://github.com/prezly/theme-nextjs-greta/assets/4209081/0445da20-561e-4fcd-898b-7452d0a61978)

Reply:
Looks good to me!

Reply:
Also, I think this comment belongs to the Lena PR, not Greta :D

Reply:
Also, I think this comment belongs to the Lena PR, not Greta :D

Reply:
Ah yes, I got it mixed up 😄